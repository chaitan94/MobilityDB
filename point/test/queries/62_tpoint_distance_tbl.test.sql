-------------------------------------------------------------------------------

SELECT round(g <-> temp, 6) FROM tbl_geompoint t1, tbl_tgeompoint t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(temp <-> g, 6) FROM tbl_tgeompoint t1, tbl_geompoint t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(t1.temp <-> t2.temp, 6) FROM tbl_tgeompoint t1, tbl_tgeompoint t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;

SELECT round(round(g <-> temp, 6), 6) FROM tbl_geogpoint t1, tbl_tgeogpoint t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(round(temp <-> g, 6), 6) FROM tbl_tgeogpoint t1, tbl_geogpoint t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(round(t1.temp <-> t2.temp, 6), 6) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;

-------------------------------------------------------------------------------

SELECT round(g <-> temp, 6) FROM tbl_geompoint3D t1, tbl_tgeompoint3D t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(temp <-> g, 6) FROM tbl_tgeompoint3D t1, tbl_geompoint3D t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(t1.temp <-> t2.temp, 6) FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;

SELECT round(round(g <-> temp, 6), 6) FROM tbl_geogpoint3D t1, tbl_tgeogpoint3D t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10; 
SELECT round(round(temp <-> g, 6), 6) FROM tbl_tgeogpoint3D t1, tbl_geogpoint3D t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
SELECT round(round(t1.temp <-> t2.temp, 6), 6) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;

-------------------------------------------------------------------------------

	

