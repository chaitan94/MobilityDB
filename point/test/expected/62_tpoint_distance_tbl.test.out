SELECT g <-> temp FROM tbl_geompoint t1, tbl_tgeompoint t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                 
-----------------------------------------
 6.6776001089177@2001-08-06 05:33:00+00
 64.641459816143@2001-05-09 14:16:00+00
 70.4011464088226@2001-06-11 14:33:00+00
 32.5791523703265@2001-12-03 12:05:00+00
 59.9308743603429@2001-12-03 12:05:00+00
 12.608057278247@2001-08-06 05:33:00+00
 70.4011464088226@2001-06-11 14:33:00+00
 32.585733684272@2001-08-11 06:38:00+00
 32.585733684272@2001-08-11 06:38:00+00
 30.5908568423842@2001-10-09 17:44:00+00
(10 rows)

SELECT temp <-> g FROM tbl_tgeompoint t1, tbl_geompoint t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                 
-----------------------------------------
 59.4115758659539@2001-06-11 14:33:00+00
 100.37240631917@2001-06-11 14:33:00+00
 84.4163551876945@2001-06-11 14:33:00+00
 6.02525163644672@2001-06-11 14:33:00+00
 8.71001160900317@2001-06-11 14:33:00+00
 110.620600839645@2001-06-11 14:33:00+00
 95.2875639907718@2001-06-11 14:33:00+00
 78.3231480682677@2001-06-11 14:33:00+00
 37.9943353523685@2001-06-11 14:33:00+00
 73.126222100656@2001-06-11 14:33:00+00
(10 rows)

SELECT t1.temp <-> t2.temp FROM tbl_tgeompoint t1, tbl_tgeompoint t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                 
-----------------------------------------
 0@2001-06-11 14:33:00+00
 0@2001-06-11 14:33:00+00
 0@2001-08-11 06:38:00+00
 0@2001-08-11 06:38:00+00
 0@2001-06-11 14:33:00+00
 0@2001-06-11 14:33:00+00
 0@2001-08-11 06:38:00+00
 0@2001-08-11 06:38:00+00
 28.4768486799868@2001-12-03 12:05:00+00
 0@2001-12-03 12:05:00+00
(10 rows)

SELECT round(g <-> temp, 6) FROM tbl_geogpoint t1, tbl_tgeogpoint t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                 round                 
---------------------------------------
 1778593.298941@2001-02-15 16:33:00+00
 2638236.28571@2001-10-16 15:01:00+00
 1778593.298941@2001-02-15 16:33:00+00
 363908.803368@2001-05-29 06:20:00+00
 1577785.650396@2001-10-03 06:00:00+00
 708510.174582@2001-05-29 06:20:00+00
 989072.816987@2001-10-03 06:00:00+00
 1331896.828904@2001-04-15 09:53:00+00
 2142168.430612@2001-12-09 09:19:00+00
 2638236.28571@2001-10-16 15:01:00+00
(10 rows)

SELECT round(temp <-> g, 6) FROM tbl_tgeogpoint t1, tbl_geogpoint t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
                 round                 
---------------------------------------
 3694042.094085@2001-10-16 15:01:00+00
 2717633.886015@2001-10-16 15:01:00+00
 1608817.161093@2001-10-16 15:01:00+00
 886439.946006@2001-10-16 15:01:00+00
 2853551.187142@2001-10-16 15:01:00+00
 3332827.792689@2001-10-16 15:01:00+00
 2974785.45351@2001-10-16 15:01:00+00
 2238314.007111@2001-10-16 15:01:00+00
 1809027.5398@2001-10-16 15:01:00+00
 2638236.28571@2001-10-16 15:01:00+00
(10 rows)

SELECT round(t1.temp <-> t2.temp, 6) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                round                 
--------------------------------------
 0@2001-10-16 15:01:00+00
 0@2001-10-16 15:01:00+00
 0@2001-02-15 16:33:00+00
 0@2001-02-15 16:33:00+00
 0@2001-10-16 15:01:00+00
 0@2001-10-16 15:01:00+00
 0@2001-02-15 16:33:00+00
 0@2001-02-15 16:33:00+00
 503065.965736@2001-05-29 06:20:00+00
 0@2001-05-29 06:20:00+00
(10 rows)

SELECT g <-> temp FROM tbl_geompoint3D t1, tbl_tgeompoint3D t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                 
-----------------------------------------
 80.9031313478982@2001-06-10 14:03:00+00
 40.4506828487824@2001-07-18 13:10:00+00
 77.4733134448818@2001-01-01 05:13:00+00
 64.5164808316712@2001-05-21 10:03:00+00
 62.9125315603757@2001-05-21 10:03:00+00
 73.1275400803282@2001-06-10 14:03:00+00
 77.4733134448818@2001-01-01 05:13:00+00
 20.7485278373381@2001-11-05 01:52:00+00
 20.7485278373381@2001-11-05 01:52:00+00
 70.7754195873975@2001-04-26 00:13:00+00
(10 rows)

SELECT temp <-> g FROM tbl_tgeompoint3D t1, tbl_geompoint3D t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                 
-----------------------------------------
 63.6632505677035@2001-01-01 05:13:00+00
 75.4843545706177@2001-01-01 05:13:00+00
 83.487225435849@2001-01-01 05:13:00+00
 75.638465317209@2001-01-01 05:13:00+00
 82.3092068545377@2001-01-01 05:13:00+00
 92.680293114993@2001-01-01 05:13:00+00
 102.76545880319@2001-01-01 05:13:00+00
 75.4109542691363@2001-01-01 05:13:00+00
 98.3239710471906@2001-01-01 05:13:00+00
 94.4769218060272@2001-01-01 05:13:00+00
(10 rows)

SELECT t1.temp <-> t2.temp FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                ?column?                
----------------------------------------
 0@2001-01-01 05:13:00+00
 0@2001-01-01 05:13:00+00
 0@2001-11-05 01:52:00+00
 0@2001-11-05 01:52:00+00
 0@2001-01-01 05:13:00+00
 0@2001-01-01 05:13:00+00
 0@2001-11-05 01:52:00+00
 0@2001-11-05 01:52:00+00
 50.834283973891@2001-05-21 10:03:00+00
 0@2001-05-21 10:03:00+00
(10 rows)

SELECT round(g <-> temp, 6) FROM tbl_geogpoint3D t1, tbl_tgeogpoint3D t2
WHERE g <-> temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                 round                 
---------------------------------------
 2927529.504995@2001-08-30 21:53:00+00
 3446929.778077@2001-06-22 10:37:00+00
 2927529.504995@2001-08-30 21:53:00+00
 931127.091299@2001-03-25 21:59:00+00
 1968329.864975@2001-11-07 18:54:00+00
 975467.508146@2001-03-25 21:59:00+00
 3465948.124914@2001-11-07 18:54:00+00
 763191.410349@2001-12-23 08:21:00+00
 1715988.894693@2001-10-10 18:35:00+00
 3446929.778077@2001-06-22 10:37:00+00
(10 rows)

SELECT round(temp <-> g, 6) FROM tbl_tgeogpoint3D t1, tbl_geogpoint3D t2
WHERE temp <-> g IS NOT NULL ORDER BY t1.k LIMIT 10;
                 round                 
---------------------------------------
 1301865.723548@2001-06-22 10:37:00+00
 1478444.519823@2001-06-22 10:37:00+00
 2177447.467553@2001-06-22 10:37:00+00
 3506000.435895@2001-06-22 10:37:00+00
 4010409.404335@2001-06-22 10:37:00+00
 1080848.343982@2001-06-22 10:37:00+00
 3309009.389959@2001-06-22 10:37:00+00
 3991731.128601@2001-06-22 10:37:00+00
 1932528.697964@2001-06-22 10:37:00+00
 3446929.778077@2001-06-22 10:37:00+00
(10 rows)

SELECT round(t1.temp <-> t2.temp, 6) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2
WHERE t1.temp <-> t2.temp IS NOT NULL ORDER BY t1.k LIMIT 10;
                round                 
--------------------------------------
 0@2001-06-22 10:37:00+00
 0@2001-06-22 10:37:00+00
 0@2001-08-30 21:53:00+00
 0@2001-08-30 21:53:00+00
 0@2001-06-22 10:37:00+00
 0@2001-06-22 10:37:00+00
 0@2001-08-30 21:53:00+00
 0@2001-08-30 21:53:00+00
 696064.597846@2001-03-25 21:59:00+00
 0@2001-03-25 21:59:00+00
(10 rows)

